lib/network/kmip/Client.js

```diff
@@ -387,18 +407,17 @@ class Client {
                 logger.error('KMIP::_revokeBucketKey',
                              { error: err,
                                serverInformation: this.serverInformation });
-                return cb(err);
+                return cb(_arsenalError(err));
             }
             const uniqueIdentifier =
                   response.lookup(searchFilter.uniqueIdentifier)[0];
             if (uniqueIdentifier !== bucketKeyId) {
-                const error = Error(
-                    'Server did not return the right identifier',
-                    { uniqueIdentifier });
+                const error = 'Server did not return the expected identifier';
                 logger.error('KMIP::_revokeBucketKey',
                              { error, uniqueIdentifier });
+                return cb(_arsenalError(error));
             }
-            return cb(err);
+            return cb(null);
```

Comment:
`return cb();` We don't have to explicitly set nulls for errors if we don't need it :)